components/homme/src/share/namelist_mod.F90

```diff
@@ -7,6 +7,7 @@ module namelist_mod
   use kinds,      only: real_kind, iulog
   use params_mod, only: recursive, sfcurve, SPHERE_COORDS, Z2_NO_TASK_MAPPING
   use cube_mod,   only: rotate_grid
+  use dyn_grid,   only: fv_nphys
```

Comment:
This and the next new line here should probably be wrapped in #ifndef CAM .

Reply:
Is that for standalone HOMME tests?

Reply:
yes

Reply:
ok, I'll add this.